docs/source/operations_guide.rst

```diff
@@ -131,12 +131,12 @@ certificates for peers and orderers.
 
 You will issue the commands below to enroll the TLS CA admin and then register
 identities. We assume the trusted root certificate for the TLS CA has been copied
-to ``/tmp/hyperledger/tls-ca/crypto/tls-ca-cert.pem`` on all host machines that
+to ``/tmp/hyperledger/tls-ca/crypto/tls-cert.pem`` on all host machines that
```

Comment:
I think the suggestion is to make the file name "tls-ca-cert.pem" (instead of "tls-cert.pem") so that you know that this is a TLA CA cert and not a regular TLS cert. Therefore I'm not sure we should make this change.